tools/dashboard-plugin-tests/package.json

```diff
@@ -0,0 +1,154 @@
+{
+    "name": "@gooddata/dashboard-plugin-tests",
+    "version": "1.0.0",
+    "description": "Application and servers for dashboard plugins API testing",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/gooddata/gooddata-ui-sdk.git",
+        "directory": "tools/dashboard-plugin-tests"
+    },
+    "author": "GoodData Corporation",
+    "license": "MIT",
```

Comment:
This must not be MIT since it uses the sdk-ui-charts. Please use the same LICENCE text and spec as sdk-ui-charts.